codalab/rest/util.py

```diff
@@ -57,7 +58,9 @@ def wrapper(*args, **kwargs):
             # Request context not initialized: we are NOT in a Bottle app
             # Fabricate a thread-local context for LocalBundleClient
             if client is not None:
-                user_id = client._current_user_id()
+                user_id = (client._current_user_id()
```

Comment:
When would _current_user_id() not exist?  Document this and whether this is temporary and will go away when everything is REST-ified.